backend/penndata/management/commands/get_engineering_events.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import html
+import json
+
+import requests
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from penndata.models import Event
+
+
+ENGINEERING_EVENTS_WEBSITE = "https://events.seas.upenn.edu/calendar/list/"
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        try:
+            resp = requests.get(ENGINEERING_EVENTS_WEBSITE)
+        except ConnectionError:
+            print("Error:", ConnectionError)
+            return None
+
+        html_content = resp.text
+
+        start_marker = '<script type="application/ld+json">'
+        end_marker = "</script>"
+        start_index = html_content.find(start_marker)
+        end_index = html_content.find(end_marker, start_index)
+        json_ld_content = html_content[start_index + len(start_marker) : end_index]
+
+        events_data = json.loads(json_ld_content)
+
+        for event in events_data:
+            event_name = html.unescape(event.get("name", ""))
```

Comment:
If `event_name` here is blank, do you think we should still create the `Event` object? I feel like the name is necessary